### PR TITLE
fix(onboarding): Correct flex sizing of getting-started boxes

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Comment by ckj** [comment]
@EvanPurkhiser mind attaching a screenshot?

**Comment by EvanPurkhiser** [comment]
Check percy @ckj

File: src/sentry/static/sentry/app/views/projectInstall/gettingStarted.jsx [hunk 1]
```diff
@@ -14,7 +15,7 @@ class GettingStartedBody extends React.Component {
   render() {
     let {project, organization} = this.context;
     return (
-      <div className="getting-started">
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectInstall/gettingStarted.jsx]: did we check to see if this was used anywhere?

**Review by billyvg — COMMENTED** [review]
Did we used to have a sidebar on the right?

File: src/sentry/static/sentry/app/views/projectInstall/gettingStarted.jsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/projectInstall/gettingStarted.jsx]: The getting-started class wasn't even declared anywhere.

Comment (markstory) [reply src/sentry/static/sentry/app/views/projectInstall/gettingStarted.jsx]: The only hits are in the red side of this diff.

**Review by markstory — APPROVED** [review]

**Pull request merged**